lib/constants.js

```diff
@@ -2,6 +2,8 @@
 
 const TOKEN_BEARER_KEY_NAME = 'Authorization';
```

Comment:
To make it pretty, can you align all '=' with spaces. So they are in a single column and they are easier to read.